lib/cc/module_meta.rb

```diff
@@ -95,6 +95,7 @@ def create_module_meta(document=nil)
                   item_node.position ct.position
                   item_node.new_tab ct.new_tab
                   item_node.indent ct.indent
+                  item_node.part_id ct.part_id
```

Comment:
Is this part of the magic field fix?  Or partial work from the dynamic syllabus work?